web/src/components/dashboard/anytime-actions/AnytimeActionDropdown.tsx

```diff
@@ -33,45 +33,59 @@ export const AnytimeActionDropdown = (props: Props): ReactElement => {
   const industryId = business?.profileData.industryId;
   const sectorId = business?.profileData.sectorId;
 
+  const getApplicableAnytimeActions = (): AnytimeActionWithType[] => {
```

Comment:
Pulled this into a helper in case we want to pull out this logic to its own file later but decided to keep it here for now

Reply:
I will always support refactors like this, regardless of whether or not it does get reused, since it makes reading the code easier. Well done.